### ISSUE TITLE
Role fails with existing pyenv installation

### ISSUE THREAD

**Issue by mdklatt** [mdklatt/ansible-python3-role#7]
The [pyenv installer](https://github.com/pyenv/pyenv-installer) now fails if it detects an existing installation, which causes the role to fail:

``` 
TASK [python3 : run pyenv installer] *******************************************
task path: python3-role/tasks/pyenv.yml:12
fatal: [centos]: FAILED! => {"changed": true, "cmd": ["bash", "pyenv-installer"], "delta": "0:00:00.006871", "end": "2019-02-23 21:57:48.493896", "msg": "non-zero return code", "rc": 1, "start": "2019-02-23 21:57:48.487025", "stderr": "\nWARNING: Can not proceed with installation. Kindly remove '.pyenv' from /home/vagrant first.", "stderr_lines": ["", "WARNING: Can not proceed with installation. Kindly remove '.pyenv' from /home/vagrant first."], "stdout": "", "stdout_lines": []}
```

**Comment by mdklatt** [comment]
The role already checks for the desired Python version before running `pyenv`, but it needs to check for `pyenv` itself before running the installer.

**Comment by mdklatt** [comment]
Commit 160c78aaa70276f76843e3268dd83075bf216631 adds a check for an existing `pyenv` using a `creates` argument. The deeper issue is that this is still vulnerable to upstream changes. Using  a local copy of the `pyenv` install script would isolate the role from breaking changes, but also from the latest updates. For now, erring on the side of keeping things up-to-date.

**Comment by mdklatt** [comment]
Issue resolved by commit 160c78aaa70276f76843e3268dd83075bf216631.
